src/python/datastore/data_handler.py

```diff
@@ -74,6 +75,9 @@
     '</b>' % (data_types.FILE_CONSISTENT_UNREPRODUCIBLE_TESTCASE_DEADLINE,
               data_types.UNREPRODUCIBLE_TESTCASE_WITH_BUG_DEADLINE))
 
+FuzzerDisplay = collections.namedtuple(
```

Comment:
Display feels a little bit ambiguous and it doesn't look like we're really using it only for display purposes. It might be fine to keep it as-is since I'm having trouble thinking of a better name (something like FuzzerInfo comes to mind but that's still ambiguous, so not really an improvement).

Reply:
Yes cant think of anything better.